lib/chef-cli/cli.rb

```diff
@@ -104,21 +104,24 @@ def show_version
     end
 
     def show_version_via_version_manifest
-      msg("#{ChefCLI::Dist::PRODUCT} version: #{manifest_field("build_version")}")
+      msg("#{ChefCLI::Dist::PRODUCT} version: #{component_version("build_version")}")
+
       { "#{ChefCLI::Dist::INFRA_CLIENT_PRODUCT}": "#{ChefCLI::Dist::INFRA_CLIENT_GEM}",
         "#{ChefCLI::Dist::INSPEC_PRODUCT}": "#{ChefCLI::Dist::INSPEC_CLI}",
         "#{ChefCLI::Dist::CLI_PRODUCT}": "#{ChefCLI::Dist::CLI_GEM}",
+        "#{ChefCLI::Dist::HAB_PRODUCT}": "habitat",
```

Comment:
```suggestion
        "#{ChefCLI::Dist::HAB_PRODUCT}": "#{ChefCLI::Dist::HAB_SOFTWARE_NAME}",
```

What do you think about `HAB_SOFTWARE_NAME` as in 'Omnibus Software Definition'? I also thought about `HAB_PRODUCT_ID` or `HAB_PRODUCT_SHORT`.

Reply:
Though I think a better long-term fix might be change the naming to line up something like `THING_PRODUCT_TOKEN`,  (habitat) `THING_PRODUCT_DESCRIPTION (Chef Habitat) , THING_CLI (hab). `.  There's a bit of inconsistency now that I think that would help with.